language-support/hs/bindings/test/DA/Ledger/Tests.hs

```diff
@@ -697,71 +700,34 @@ assertTextContains text frag =
     unless (frag `isInfixOf` text) (assertFailure msg)
     where msg = "expected frag: " ++ frag ++ "\n contained in: " ++ text
 
-----------------------------------------------------------------------
--- test with/out shared sandboxes...
-
 darPath :: FilePath
 darPath = mainWorkspace </> "language-support/hs/bindings/for-tests.dar"
 
-createSpec :: Maybe AuthSpec -> IO SandboxSpec
-createSpec maybeAuth = do
-    dar <- locateRunfiles darPath
-    return SandboxSpec {dar, maybeAuth}
-
-data ShareSandbox = ShareSandbox Bool (Maybe AuthSpec)
-
-testGroupWithSandbox :: ShareSandbox -> TestName -> [WithSandbox -> TestTree] -> TestTree
-testGroupWithSandbox (ShareSandbox enableSharing maybeAuth) name tests = do
-    if enableSharing
-    then
-        -- waits to run in the one shared sandbox
-        withResource (acquireShared maybeAuth) releaseShared $ \resource -> do
-        testGroup name $ map (\f -> f (withShared maybeAuth resource)) tests
-    else do
-        -- runs in it's own freshly (and very slowly!) spun-up sandbox
-        let withSandbox' f = do
-                spec <- createSpec maybeAuth
-                darMetadata <- getDarMetadata spec
-                withSandbox spec $ \sandbox -> f sandbox maybeAuth darMetadata (TestId 0)
-        testGroup name $ map (\f -> f withSandbox') tests
-
-getDarMetadata :: SandboxSpec -> IO DarMetadata
-getDarMetadata SandboxSpec{dar} = do
+testGroupWithSandbox :: FilePath -> Maybe Secret -> TestName -> [WithSandbox -> TestTree] -> TestTree
```

Comment:
I just killed the non-shared part from the tests here. We haven’t used it for ages and I have absolutely no intention of bringing it back.